MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -6242,6 +6242,12 @@ public string GetProgressiveAudioFullCommandLine(EncodingJobInfo state, Encoding
                 audioTranscodeParams.Add("-acodec " + GetAudioEncoder(state));
             }
 
+            if (GetAudioEncoder(state).StartsWith("pcm_", StringComparison.Ordinal))
+            {
+                audioTranscodeParams.Add(string.Concat("-f ", GetAudioEncoder(state).Substring(4)));
```

Comment:
```suggestion
                audioTranscodeParams.Add("-f " + GetAudioEncoder(state).Substring(4));
```

Reply:
This patch will cause compile warning:

Error: /home/runner/work/jellyfin/jellyfin/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs(6247,42): error CA1845: Use span-based 'string.Concat' and 'AsSpan' instead of 'Substring' 
(https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1845) [/home/runner/work/jellyfin/jellyfin/MediaBrowser.Controller/MediaBrowser.Controller.csproj]

So I think use `string.Concat` is correct

Reply:
Yeak, looks like the compiler prefers `string.Concat` because you can use 'AsSpan' instead of 'Substring', see my new suggestion